airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -71,6 +71,9 @@ def __init__(
         aws_conn_id: str = "aws_default",
         steps: list[dict] | str | None = None,
         wait_for_completion: bool = False,
+        cancel_existing_steps: bool = True,
+        steps_states: list[str],
+        cancellation_option: str = "SEND_INTERRUPT",
```

Comment:
Can you update the docstring above with these new params

Reply:
Added docstring for all the implemented methods and special case conditional

Please point me after going through my updated commit, if I missed it somewhere and needs to be added